app/views/forecasts/uploads/update.html.haml

```diff
@@ -1,20 +1,28 @@
 = content_for :page_title_prefix, t("page_title.forecast.upload")
 
 %main.govuk-main-wrapper#main-content{ role: "main" }
-  .govuk-grid-row
-    - if @success
-      .govuk-grid-column-full
-        %h1.govuk-heading-xl
-          = t("importer.success.heading")
-        = render partial: "shared/forecasts/forecasts_by_activity"
-        = link_to t("importer.success.back_link"), report_forecasts_path(@report_presenter), class: "govuk-button"
-
-    - else
+  - unless @success
+    .govuk-grid-row
```

Comment:
Is there a reason for structuring it this way versus putting `if @success` first?

Reply:
I think we wanted to deal with the "unhappy" path first. If it's consistent across the various pages, it's worth the cognitive discomfort of the logic pretzel. If it's not consistent, you can make a case for easier to read logic. :)

Reply:
Seems like it's not 😅

<img width="380" alt="image" src="https://user-images.githubusercontent.com/40244233/224766764-71e3b2dc-b8f0-40b0-9883-ef6ec63cd5be.png">

Reply:
If we start looking at inconsistencies, we also notice that some pages have a different page title for success vs failure, which contributes to the meandering conditionals 😬